app/exceptions/register.py

```
@@ -0,0 +1,28 @@
+    app = FastAPI()
+    register_exception_handlers(app)
+"""
+
+from fastapi import FastAPI
+
+from app.exceptions.custom_exceptions import DevDoxAPIException
+from app.exceptions.handlers import devdox_base_exception_handler, generic_exception_handler
+
+
+def register_exception_handlers(app: FastAPI):
+	app.add_exception_handler(Exception, generic_exception_handler)
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)